src/main/resources/application.properties

```diff
@@ -1 +1,10 @@
 spring.application.name=rc-manager
```

Comment:
do not store application.properties file in git. Instead, make application.properties.sample file and store it